From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Sun, 27 Sep 2020 11:04:55 +0100
Subject: DESTDIR

Make go.sh listen to DESTDIR.

Add shebang to make go.sh into a proper shell script.

Set option so go.sh will terminate on an intermediate error.
---
 go.sh | 15 ++++++++++-----
 1 file changed, 10 insertions(+), 5 deletions(-)

diff --git a/go.sh b/go.sh
index 7e5e626..beaaad2 100755
--- a/go.sh
+++ b/go.sh
@@ -1,3 +1,7 @@
+#!/bin/sh
+
+set -e
+
 # TODO: Helper script - automate the mundane !
 
 # This script should do several things.
@@ -27,16 +31,17 @@ VERSION=v1.5
 echo "Installing Intel firmware and topology $VERSION to $INTEL_PATH"
 
 # wipe previous releases
-rm -rf ${ROOT}/${INTEL_PATH}/sof/*
-rm -rf ${ROOT}/${INTEL_PATH}/sof-tplg-*
-rm -rf ${ROOT}/${INTEL_PATH}/sof-tplg
+rm -rf ${DESTDIR}${ROOT}/${INTEL_PATH}/sof/*
+rm -rf ${DESTDIR}${ROOT}/${INTEL_PATH}/sof-tplg-*
+rm -rf ${DESTDIR}${ROOT}/${INTEL_PATH}/sof-tplg
 
 # copy to destination
 cd lib/firmware
-cp -rf intel ${ROOT}/lib/firmware
+mkdir -p ${DESTDIR}${ROOT}/lib/firmware
+cp -rf intel ${DESTDIR}${ROOT}/lib/firmware
 
 # add symlinks
-cd ${ROOT}/${INTEL_PATH}/sof
+cd ${DESTDIR}${ROOT}/${INTEL_PATH}/sof
 
 ln -s ${VERSION}/sof-bdw-${VERSION}.ri sof-bdw.ri
 ln -s ${VERSION}/sof-byt-${VERSION}.ri sof-byt.ri
